lib/batch.js

```diff
@@ -33,6 +33,10 @@ module.exports.config = function (settings) {
 
             request.payload.requests.every((req, idx) => {
 
+                if (req.query) {
+                    req.payload = req.query;
```

Comment:
We should not convert query parameters into request payloads in order to pipeline data into query parameters.

Bassmaster should look at the request it's processing, and if it contains a query object, it should see if previous request responses need to be pipelined into it before constructing a URL and injecting it.

Reply:
@cadecairos Hello Chris,
I have made another commit. Please have a look at it. Just how it was desired. Bassmaster looks at the request it is processing, and it the request contains a query object, it sees if the previous request responses needs to be pipelined into it before constructing a URL and injecting it. Wrote a test case too to cover the same. 

Please review and let me know if anything else also needs to be done.

Thankyou,
Aditya